site/source/user-manual/connecting/connecting-lan/lan-browser/lan-chrome-brave.rst

```diff
@@ -1,51 +1,59 @@
-.. _lan-brave:
+.. _lan-chrome-brave:
 
 =========================
-Trust Embassy CA in Brave
+Trust Embassy CA in Chrome / Brave Browser
 =========================
```

Comment:
It's dumb, but these `====` need to be extended to end of line at top and bottom or we'll get RST warnings in our builds